### PR TITLE
updates, tests, for new jenkins log annotations

### PR DESCRIPTION
**Pull request by gabemontero**
@openshift/devex PTAL

### CODE REVIEW

**Comment by gabemontero** [comment]
[testextended][extended:core(Pipeline log annotation check)]

File: test/extended/builds/pipeline.go [hunk 1]
```diff
@@ -228,6 +228,32 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
...
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// start the build
+			g.By("starting the pipeline build")
+			br, _ := exutil.StartBuildAndWait(oc, "sample-pipeline-withenvs")
+			// note, this will check the annotations if there is a build failure
+			debugAnyJenkinsFailure(br, oc.Namespace()+"-sample-pipeline-withenvs", oc, true)
+			br.AssertSuccess()
+
+			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
```
Comment (bparees) [inline test/extended/builds/pipeline.go]: can't we just check for these in one of the other tests that's already running a pipeline build?

Comment (gabemontero) [inline test/extended/builds/pipeline.go]: update pushed

**Comment by bparees** [comment]
lgtm pending extended test pass

**Comment by gabemontero** [comment]
[testextended][extended:core(Pipeline with env vars)]

**Comment by bparees** [comment]
[merge]

**Pull request merged**